services/avs/mgmt/2020-03-20/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+pub mod models;
+pub mod operations;
+
+pub const API_VERSION: &str = "2020-03-20";
+
+pub type Error = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, Error>;
+
+pub struct Configuration {
+    pub api_version: String,
+    pub client: reqwest::Client,
+    pub base_path: String,
+    pub bearer_access_token: Option<String>,
+}
```

Comment:
It would be nice to include a `Configuration::new` function which just takes a access token `String` and defaults all other fields.

Reply:
Done. Using `Configuration` is what openapi-generator does. I'm wondering if we wish to try creating a Client instead with operations exposed on it like other AutoRest languages. I think this is going to undergo a few interactions before we get it right.

Reply:
I've opened an issue for creating a client abstraction https://github.com/Azure/azure-sdk-for-rust/issues/36